src/platform/orchestrator/src/MpiServer.cpp

```diff
@@ -115,28 +101,97 @@ std::string CreateGUID()
     return ss.str();
 }
 
-void MpiApiInitialize()
+std::string StatusText(StatusCode statusCode)
 {
-    PlatformLog::OpenLog();
-    server.Listen();
+    std::string result;
+    switch (statusCode)
+    {
+        case StatusCode::OK:
+            result = "OK";
+            break;
+        case StatusCode::BAD_REQUEST:
+            result = "BAD_REQUEST";
+            break;
+        case StatusCode::NOT_FOUND:
+            result = "NOT_FOUND";
+            break;
+        default:
+            result = "INTERNAL_SERVER_ERROR";
```

Comment:
These can change to "ok", "bad request", "not found", etc. If used for logging.

Reply:
Not using these for logging, only for creating the response header.